server/plugin/auth_token_test.go

```diff
@@ -0,0 +1,169 @@
+package plugin
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"encoding/json"
+	"errors"
+	"io"
+	"testing"
+
+	"bou.ke/monkey"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/oauth2"
+)
+
+type mockBLock struct{}
+
+func (b *mockBLock) BlockSize() int { return 1 }
```

Comment:
any reason to return 1 because default value for int is 0

Reply:
No specific reason, changed to 0